modules/virt-configuring-a-live-migration-policy.adoc

```diff
@@ -36,8 +68,7 @@ spec:
       hpc-workloads: "True"
       xyz-workloads-type: ""
     virtualMachineInstanceSelector: <2>
-      workload-type: "db"
-      operating-system: ""
+      "kubevirt.io/environment": "production"
```

Comment:
Parentheses required here? I do not see them used in the other example.

Reply:
AFAIK it should work either way, but you're right that for consistency, it'd be better if they weren't there.